repl/src/test/scala/org/apache/spark/repl/ReplSuite.scala

```diff
@@ -290,6 +290,9 @@ class ReplSuite extends SparkFunSuite with BeforeAndAfterAll {
         |# Set the log level for this class to ERROR same as the default setting.
         |logger.repl.name = org.apache.spark.repl.Main
         |logger.repl.level = error
+        |
+        |logger.customLogger2.name = customLogger2
+        |logger.customLogger2.level = info
```

Comment:
Same as `LoggingSuite`. If we want to have loggers with different level than repl's one, we need to have custom configuration for them in configuration file explicitly.